src/index.html

```diff
@@ -6,7 +6,39 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page">
+    <section class="calendar calendar--start-day--sun calendar--month-length--29">
+      <div class="calendar__day calendar__day--date--1"></div>
```

Comment:
don't create separate class with date number for each day
use nth-child instead of separate class in css